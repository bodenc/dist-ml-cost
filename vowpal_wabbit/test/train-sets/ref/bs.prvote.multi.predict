1.000000 0.997296 1.296013
0.000000 -0.024881 0.170573
3.000000 1.818539 2.858242
0.000000 -0.143870 0.338941
0.000000 -0.095287 0.146111
2.000000 1.812118 2.769007
0.000000 -0.114057 0.118428
0.000000 -0.182801 0.013014
3.000000 2.546088 3.000000
1.000000 0.381745 0.532782
0.000000 0.243442 0.290122
3.000000 0.514161 2.873086
0.000000 0.010703 0.138818
0.000000 -0.102617 0.241717
2.000000 1.773769 2.113003
1.000000 0.879776 0.972381
1.000000 0.881054 1.215401
2.000000 2.035578 2.444676
0.000000 0.090615 0.678673
0.000000 0.195989 0.416920
2.000000 0.572330 2.175719
1.000000 0.948689 1.426823
0.000000 -0.204115 0.754271
2.000000 1.976621 2.535813
0.000000 -0.228134 -0.001704
0.000000 0.026379 0.416575
3.000000 2.539795 3.000000
0.000000 -0.081224 0.289972
1.000000 0.583283 1.102605
1.000000 0.657689 0.770682
1.000000 0.888209 0.976421
0.000000 0.044036 0.394508
2.000000 2.322996 2.994010
0.000000 0.045135 0.300119
1.000000 0.958525 1.093620
3.000000 2.827745 3.000000
1.000000 0.747264 1.063062
0.000000 -0.233451 0.062793
2.000000 1.754919 2.864041
1.000000 1.280505 1.389261
0.000000 -0.054453 0.070220
2.000000 2.084350 2.498975
0.000000 -0.150979 0.059899
0.000000 0.024889 0.836689
2.000000 1.708614 2.601858
0.000000 -0.097144 0.498147
1.000000 0.022822 0.647841
2.000000 2.233139 2.637181
1.000000 0.836292 0.981192
1.000000 0.543057 0.721317
2.000000 1.957688 2.779601
1.000000 0.805914 0.989266
3.000000 2.725823 2.850785
0.000000 -0.140018 0.444593
1.000000 1.091665 1.485385
3.000000 2.198494 2.856077
0.000000 -0.227155 -0.064318
2.000000 1.818663 2.761198
1.000000 0.709630 0.989899
1.000000 0.638766 0.754106
0.000000 -0.606652 0.068781
0.000000 -0.191876 0.093611
1.000000 0.792914 0.967395
0.000000 -0.181684 -0.027351
-1.000000 -0.892949 -0.585002
0.000000 -0.201341 -0.095313
0.000000 -0.128911 0.139969
-1.000000 -1.000000 0.483943
0.000000 -0.090407 0.242410
1.000000 0.764400 0.931225
-1.000000 -0.604451 -0.364373
1.000000 0.941816 1.142960
1.000000 0.721201 1.065755
0.000000 0.026044 0.368995
1.000000 0.606240 0.998738
1.000000 0.954040 1.026910
0.000000 -0.137300 0.106709
0.000000 -0.170735 0.030993
0.000000 -0.036332 0.069993
1.000000 0.834524 1.054736
0.000000 -0.147490 0.080111
0.000000 -0.092213 0.045656
1.000000 0.983203 1.133457
-1.000000 -0.991592 -0.761992
0.000000 -0.164116 0.017370
0.000000 -0.090992 0.067626
1.000000 0.936973 1.137832
1.000000 0.911019 1.096976
0.000000 -0.896316 -0.166576
1.000000 0.598708 0.781356
0.000000 -0.017629 0.119009
1.000000 0.933704 0.984708
1.000000 0.859511 1.274853
0.000000 -0.055805 0.013013
0.000000 -0.684304 -0.145759
0.000000 -0.130465 0.057897
1.000000 0.903859 1.075045
0.000000 0.006028 0.132119
1.000000 1.070429 1.178548
1.000000 0.557164 0.740026
0.000000 0.012004 0.306633
1.000000 0.813142 1.039797
0.000000 0.219353 1.045170
-1.000000 -0.918504 -0.120672
0.000000 -0.017080 0.275518
1.000000 0.949006 1.107581
0.000000 -0.072708 0.047928
0.000000 -0.081827 0.117147
3.000000 2.162519 2.963920
1.000000 0.617064 0.745980
1.000000 0.954738 1.051530
2.000000 0.837229 2.520384
0.000000 -0.078817 0.063057
1.000000 0.871075 1.264038
0.000000 0.118992 0.615175
0.000000 -0.085065 0.025072
2.000000 0.431598 2.093800
1.000000 0.941992 1.273427
1.000000 0.764991 0.966257
0.000000 0.409039 0.572215
0.000000 -0.081031 0.012337
1.000000 0.986233 1.035113
3.000000 1.696929 2.823445
1.000000 0.922767 1.197439
1.000000 0.993362 1.158937
1.000000 0.356116 0.861828
0.000000 -0.075009 0.052994
1.000000 0.760490 1.083998
0.000000 -0.086802 0.062083
1.000000 0.561948 0.814420
0.000000 0.104884 0.209097
1.000000 0.849945 1.048296
0.000000 0.106123 0.168837
1.000000 0.914669 1.033538
3.000000 1.930148 2.896898
0.000000 -0.083969 0.125262
1.000000 0.876787 1.080578
1.000000 0.935528 0.970666
1.000000 1.140657 1.348179
0.000000 0.281447 0.419381
0.000000 0.035235 0.396061
0.000000 -0.093786 0.196600
1.000000 1.096046 1.159018
2.000000 1.898286 2.456885
1.000000 0.831951 0.905605
1.000000 0.650382 0.916158
1.000000 0.749956 1.184740
2.000000 1.960806 2.448737
0.000000 -0.147815 0.580457
1.000000 0.882084 1.055152
1.000000 0.606088 0.920796
3.000000 2.855120 3.000000
1.000000 0.561854 0.946930
0.000000 0.148620 0.228379
3.000000 2.760775 2.901104
1.000000 0.675480 0.927743
1.000000 0.778486 1.035339
3.000000 2.323100 2.851998
0.000000 0.329307 0.665108
1.000000 0.799572 0.933874
0.000000 -0.008269 0.375510
0.000000 -0.328219 0.009213
0.000000 -0.229804 -0.116273
1.000000 -0.419678 1.121495
0.000000 -0.032790 0.140288
1.000000 0.951496 1.019486
1.000000 0.728512 1.002046
0.000000 0.090012 0.720025
-1.000000 -1.000000 -0.758333
1.000000 0.900743 1.112997
1.000000 0.910205 1.078144
0.000000 -0.029108 0.167298
0.000000 -0.169489 0.030176
1.000000 0.378798 0.906329
0.000000 0.022676 0.750413
0.000000 -0.126447 -0.016095
-1.000000 -0.906557 0.160036
1.000000 0.939241 1.008837
1.000000 0.950832 0.995436
1.000000 0.659374 0.733258
1.000000 0.929425 1.213291
0.000000 0.124843 0.279854
1.000000 0.787096 0.995988
0.000000 -0.067783 0.037388
0.000000 -0.098959 0.035617
-1.000000 -0.983385 -0.636670
1.000000 0.775490 0.982473
0.000000 0.001356 0.148851
0.000000 -0.047406 0.005275
1.000000 0.474872 1.044118
1.000000 0.890747 1.018061
0.000000 -0.007758 0.078391
0.000000 -0.003498 0.000124
0.000000 -0.053353 -0.000365
0.000000 0.009760 0.075245
1.000000 0.935274 0.999042
1.000000 0.910967 1.103930
0.000000 -0.030937 0.145855
0.000000 0.000072 0.680008
1.000000 0.448572 0.873944
