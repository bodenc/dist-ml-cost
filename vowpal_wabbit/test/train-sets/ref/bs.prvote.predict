0.000000 0.458392 1.000000
1.000000 0.452659 0.619412
0.000000 0.310507 0.456517
0.000000 0.173335 0.336450
0.000000 0.248374 0.432724
1.000000 0.935125 1.000000
0.000000 0.283262 0.428031
0.000000 0.045959 0.367023
0.000000 0.355603 0.473138
1.000000 0.949092 1.000000
0.000000 0.191053 0.379315
0.000000 0.324044 0.538324
0.000000 0.191435 0.419371
0.000000 0.310092 0.535166
1.000000 0.902808 0.994895
1.000000 0.592455 0.885632
1.000000 0.860310 1.000000
0.000000 0.057171 0.353275
0.000000 0.190818 0.644261
0.000000 0.045328 0.169037
0.000000 0.230530 0.923944
1.000000 0.611789 1.000000
0.000000 0.070341 0.310366
1.000000 0.489735 1.000000
0.000000 0.000000 0.151716
0.000000 0.138823 0.475543
0.000000 0.224302 0.449844
0.000000 0.044406 0.248419
1.000000 0.468885 0.996748
0.000000 0.243104 0.542973
1.000000 0.605581 1.000000
0.000000 0.138691 0.428629
0.000000 0.122978 0.198036
0.000000 0.272718 0.450171
1.000000 0.685426 0.997519
0.000000 0.175658 0.571002
1.000000 0.654157 1.000000
0.000000 0.099214 0.276944
1.000000 0.602038 1.000000
1.000000 1.000000 1.000000
0.000000 0.109666 0.408097
1.000000 0.826260 0.946926
0.000000 0.011032 0.125592
0.000000 0.147675 0.375113
0.000000 0.034709 0.309596
0.000000 0.083989 0.181821
0.000000 0.119489 0.701049
0.000000 0.043627 0.557042
1.000000 0.290648 0.961409
0.000000 0.222167 0.287372
1.000000 0.428921 1.000000
1.000000 0.368639 1.000000
0.000000 0.000000 0.401585
0.000000 0.092099 0.481334
1.000000 0.969464 1.000000
1.000000 0.222807 1.000000
0.000000 0.119059 0.317707
0.000000 0.069827 0.380974
0.000000 0.371971 0.885179
0.000000 0.067896 0.437237
1.000000 0.570298 1.000000
0.000000 0.016143 0.348595
1.000000 0.597393 0.885829
0.000000 0.066054 0.201453
1.000000 0.621176 1.000000
0.000000 0.093998 0.313379
0.000000 0.069193 0.460195
0.000000 0.095883 0.806659
0.000000 0.045853 0.196326
1.000000 0.566345 0.797911
0.000000 0.056480 0.425846
1.000000 0.539467 0.902147
0.000000 0.283821 0.976362
0.000000 0.000000 0.127377
1.000000 0.726537 1.000000
1.000000 0.459946 1.000000
0.000000 0.029138 0.371215
0.000000 0.187812 0.601657
0.000000 0.057713 0.119670
0.000000 0.059909 0.347788
0.000000 0.000000 0.192228
0.000000 0.141604 0.400030
1.000000 0.787472 1.000000
0.000000 0.102555 0.489685
0.000000 0.150986 0.222639
0.000000 0.117383 0.415650
0.000000 0.345909 1.000000
1.000000 0.664915 1.000000
1.000000 0.772281 1.000000
0.000000 0.000000 0.300062
0.000000 0.053952 0.209100
1.000000 0.534162 0.951390
1.000000 1.000000 1.000000
0.000000 0.072308 0.175301
1.000000 0.681809 1.000000
0.000000 0.000000 0.213567
1.000000 0.504172 1.000000
0.000000 0.027734 0.319510
1.000000 0.384518 0.950060
1.000000 0.533635 1.000000
0.000000 0.000000 0.401651
1.000000 0.534493 0.939360
0.000000 0.000000 0.104174
1.000000 0.493532 1.000000
0.000000 0.000000 0.147888
1.000000 0.963237 1.000000
0.000000 0.000000 0.143924
0.000000 0.048359 0.415193
0.000000 0.133686 0.228804
1.000000 0.917059 1.000000
1.000000 0.665322 1.000000
1.000000 0.154025 0.628948
0.000000 0.000000 0.408536
0.000000 0.317683 1.000000
0.000000 0.000000 0.509912
0.000000 0.000000 0.506966
1.000000 0.323823 0.987185
1.000000 0.452777 1.000000
0.000000 0.327834 1.000000
0.000000 0.001862 0.496234
0.000000 0.018963 0.388772
1.000000 0.927328 0.949857
0.000000 0.000000 0.404609
1.000000 0.453133 1.000000
1.000000 0.495457 0.991451
1.000000 0.369999 0.885172
0.000000 0.000000 0.405252
1.000000 0.449114 0.886408
0.000000 0.000000 0.400789
1.000000 0.414330 0.945832
0.000000 0.000000 0.117269
1.000000 0.733348 1.000000
0.000000 0.037800 0.243425
0.000000 0.237421 0.939561
0.000000 0.000000 0.367407
0.000000 0.026962 0.300478
1.000000 0.496068 1.000000
0.000000 0.350595 0.995408
1.000000 0.954148 1.000000
0.000000 0.029820 0.379896
0.000000 0.002710 0.456159
0.000000 0.000000 0.174533
1.000000 0.683822 0.909649
1.000000 0.823318 0.990685
1.000000 0.408713 0.985697
1.000000 0.611208 0.920679
1.000000 0.488054 0.945687
1.000000 0.892140 1.000000
0.000000 0.000000 0.323413
1.000000 0.309824 0.909166
1.000000 0.908086 1.000000
0.000000 0.215659 0.913836
1.000000 0.622499 0.926225
0.000000 0.000000 0.364806
1.000000 0.000000 0.663355
1.000000 0.528307 0.920641
1.000000 0.937751 1.000000
0.000000 0.000000 0.061637
1.000000 0.849636 1.000000
0.000000 0.000000 0.621811
1.000000 0.548149 0.956069
0.000000 0.000000 0.203684
0.000000 0.040461 0.379246
1.000000 0.678339 0.936413
0.000000 0.000532 0.331340
1.000000 0.409028 0.923825
1.000000 0.933654 1.000000
0.000000 0.000000 0.086990
0.000000 0.417785 0.997241
1.000000 0.705726 1.000000
1.000000 0.584154 0.913484
0.000000 0.029493 0.334615
0.000000 0.000000 0.273973
0.000000 0.314658 0.826037
0.000000 0.045142 0.200166
0.000000 0.031250 0.597995
0.000000 0.025126 0.275946
1.000000 0.774048 0.971336
1.000000 0.677971 1.000000
1.000000 0.733083 0.959720
1.000000 0.448381 0.996360
0.000000 0.083232 0.174466
1.000000 0.425506 0.908774
0.000000 0.000000 0.248872
0.000000 0.068516 0.384860
0.000000 0.000000 0.513376
1.000000 0.627244 1.000000
0.000000 0.000000 0.558645
0.000000 0.036497 0.363344
1.000000 0.671142 0.994183
1.000000 0.661582 0.920171
0.000000 0.075627 0.462856
0.000000 0.000000 0.035419
0.000000 0.000000 0.297109
0.000000 0.063500 0.297037
1.000000 0.982524 1.000000
0.000000 0.450480 0.962684
0.000000 0.000000 0.019079
0.000000 0.018797 0.458421
1.000000 0.532585 1.000000
