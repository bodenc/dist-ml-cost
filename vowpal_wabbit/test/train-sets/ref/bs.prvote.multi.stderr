only testing
Num weight bits = 18
learning rate = 10
initial_t = 1
power_t = 0.5
predictions = bs.prvote.multi.predict
using no cache
Reading datafile = train-sets/0001.multi.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.000000   0.000000            1         1.0   1.0000   1.0000       51
0.000000   0.000000            2         2.0   0.0000   0.0000      104
0.000000   0.000000            4         4.0   0.0000   0.0000      135
0.125000   0.250000            8         8.0   0.0000   0.0000      146
0.125000   0.125000           16        16.0   1.0000   1.0000       24
0.187500   0.250000           32        32.0   0.0000   0.0000       32
0.265625   0.343750           64        64.0   0.0000   0.0000       61
0.187500   0.109375          128       128.0   1.0000   1.0000      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.16
best constant = -0.00502513
total feature number = 15482
